.github/workflows/main.yml

```diff
@@ -27,7 +24,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v2
         with:
-          go-version: ${{ env.SETUP_GO_VERSION }}
+          go-version: '1.18'
```

Comment:
I dislike having the go version now buried in somewhere in the spec, versus having had to very visible at the top, before.